assets/js/modules/sign-in-with-google/components/common/Preview.js

```diff
@@ -27,6 +27,7 @@ import { __ } from '@wordpress/i18n';
  */
 import { useSelect } from 'googlesitekit-data';
 import { MODULES_SIGN_IN_WITH_GOOGLE } from '../../datastore/constants';
+import { getLocale } from '../../../../util';
```

Comment:
```suggestion
import { getLocale } from '../../../../util';
```

From the looks of it, this will use the default browser locale if no Site Kit locale is set in `global._googlesitekitLegacyData`: https://github.com/google/site-kit-wp/blob/0b1171aad556b92ecc2e847253cb3e2260f8d0d0/assets/js/util/i18n.js#L478-L500

In the case of Storybook, I can't see anywhere we actually set the locale. There are very few places we override that value from what I can see:

![Screenshot 2025-05-08 at 23 48 01](https://github.com/user-attachments/assets/0925f3cd-e1cd-468b-a6f0-86e0d3279c91)

I searched for `locale =` and `locale: ` too and couldn't find anyplace we override it in Storybook.

So it seems like we need to actually do that or we're still relying on the browser environment for Storybook as being `en-US`. 🤔

Reply:
It does fix it, on my local without this it is translating it to the Serbian, with this implementation VRT is not failing - it is tested in English

Reply:
Wild, that's so strange. Well, I guess if it works 🤷🏻‍♂️